### ISSUE TITLE
Getting Fatal Exception: java.lang.IndexOutOfBoundsException while trying to access month

### ISSUE THREAD

**Issue by credifiable** [rt2zz/react-native-contacts#282]
Device: Motorola - Moto G Turbo Edition
OS: Android 6

StackTrace:
```
Fatal Exception: java.lang.IndexOutOfBoundsException
Invalid index 1, size is 1

java.util.ArrayList.throwIndexOutOfBoundsException (ArrayList.java:255)
java.util.Arrays$ArrayList.get (Arrays.java:66)
com.rt2zz.reactnativecontacts.ContactsProvider.loadContactsFrom (ContactsProvider.java:269)
com.rt2zz.reactnativecontacts.ContactsProvider.getContacts (ContactsProvider.java:141)
com.rt2zz.reactnativecontacts.ContactsManager$1.run (ContactsManager.java:79)
android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:234)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1113)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:588)
java.lang.Thread.run (Thread.java:818)
```

https://github.com/rt2zz/react-native-contacts/blob/7bb0673883f1cc9166d04121cec720eacd85a5f4/android/src/main/java/com/rt2zz/reactnativecontacts/ContactsProvider.java#L269

**Comment by morenoh149** [comment]
Does your PR fix this issue? https://github.com/rt2zz/react-native-contacts/issues/279

**Comment by credifiable** [comment]
Hey @morenoh149 

I have encountered another Fatal Exception in my firebase crashlytics. 

```
Fatal Exception: java.lang.NumberFormatException
For input string: "Aug 7, 1980"

java.lang.Integer.parseInt (Integer.java:521)
java.lang.Integer.parseInt (Integer.java:556)
arrow_right
com.rt2zz.reactnativecontacts.ContactsProvider.loadContactsFrom (ContactsProvider.java:268)
com.rt2zz.reactnativecontacts.ContactsProvider.getContacts (ContactsProvider.java:141)
com.rt2zz.reactnativecontacts.ContactsManager$1.run (ContactsManager.java:79)
android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:243)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1133)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
java.lang.Thread.run (Thread.java:760)
```
I should create another pull request for this, wrapping the code in `try catch` and listening for this exception.

**Comment by garethmccall** [comment]
I have fixed this in PR #291
